### PR TITLE
Week 2 Tasks Completed

### PR DESCRIPTION
**Pull request by jenneft**
Ready for review.

### CODE REVIEW

File: src/JenTasks/week2/T3_consecutiveNumbers_JN.java
Comment (cbanavi) [inline src/JenTasks/week2/T3_consecutiveNumbers_JN.java:1]: lgtm

File: src/JenTasks/week2/T2_divisibleBy_3_5_15_JN.java
Comment (cbanavi) [inline src/JenTasks/week2/T2_divisibleBy_3_5_15_JN.java:1]: lgtm

File: src/JenTasks/week2/T1_swapNumbers_JN.java
Comment (cbanavi) [inline src/JenTasks/week2/T1_swapNumbers_JN.java:1]: You think you can get this to work with 0? If not, no worries, just curious.

**Review by cbanavi — CHANGES_REQUESTED** [review]
Very small change I asked, let me know.

File: src/JenTasks/week2/T1_swapNumbers_JN.java
Comment (jenneft) [reply src/JenTasks/week2/T1_swapNumbers_JN.java:1]: Let me see :)

Comment (jenneft) [reply src/JenTasks/week2/T1_swapNumbers_JN.java:1]: Adjusted formula, now it should be able to handle 0s, negatives and positives.

**Review by cbanavi — APPROVED** [review]
lgtm

**Comment by PendarShams** [comment]
LGTM

**Review by PendarShams — APPROVED** [review]

File: src/JenTasks/week2/T1_swapNumbers_JN.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package JenTasks.week2;
+
+public class T1_swapNumbers_JN {
```
[the remark this answers is not in the document]
Comment (jenneft) [reply src/JenTasks/week2/T1_swapNumbers_JN.java:3]: You need to read instructions on the project. I can assure you that it is adhering to the instructions pertaining to this project described by the Project Owner- Chiyavan.


".. Naming conventions for classes:
Create package under src folder and name it after you (ex.Chiyavan) . Then create a subpackage under your folder called week1_template or whatever the appropriate week is.
Then create your classes for the week under your subpackage (ex. finra_Chiyavan). You can see Chiyavan's branch as an example.

For naming convention; after the "classname" (the classname might have the related week's java task name or something else) you may use your name initials or etc.
Classname Examples:   Finra_Admir    or   MergeTwoArrays_AR"

File: src/JenTasks/week2/T2_divisibleBy_3_5_15_JN.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
+package JenTasks.week2;
+
+import java.util.ArrayList;
+
+public class T2_divisibleBy_3_5_15_JN {
```
[the remark this answers is not in the document]
Comment (jenneft) [reply src/JenTasks/week2/T2_divisibleBy_3_5_15_JN.java:5]: You need to read instructions on the project. I can assure you that it is adhering to the instructions pertaining to this project described by the Project Owner- Chiyavan.

".. Naming conventions for classes:
Create package under src folder and name it after you (ex.Chiyavan) . Then create a subpackage under your folder called week1_template or whatever the appropriate week is.
Then create your classes for the week under your subpackage (ex. finra_Chiyavan). You can see Chiyavan's branch as an example.

For naming convention; after the "classname" (the classname might have the related week's java task name or something else) you may use your name initials or etc.
Classname Examples: Finra_Admir or MergeTwoArrays_AR"

File: src/JenTasks/week2/T3_consecutiveNumbers_JN.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
+package JenTasks.week2;
+
+
+public class T3_consecutiveNumbers_JN {
```
[the remark this answers is not in the document]
Comment (jenneft) [reply src/JenTasks/week2/T3_consecutiveNumbers_JN.java:4]: You need to read instructions on the project. I can assure you that it is adhering to the instructions pertaining to this project described by the Project Owner- Chiyavan.

".. Naming conventions for classes:
Create package under src folder and name it after you (ex.Chiyavan) . Then create a subpackage under your folder called week1_template or whatever the appropriate week is.
Then create your classes for the week under your subpackage (ex. finra_Chiyavan). You can see Chiyavan's branch as an example.

For naming convention; after the "classname" (the classname might have the related week's java task name or something else) you may use your name initials or etc.
Classname Examples: Finra_Admir or MergeTwoArrays_AR"

File: src/JenTasks/week2/T1_swapNumbers_JN.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (cbanavi) [reply src/JenTasks/week2/T1_swapNumbers_JN.java:3]: This was my mistake by not updating the .txt file appropriately, will be sorted with future tasks

File: src/JenTasks/week2/T2_divisibleBy_3_5_15_JN.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (cbanavi) [reply src/JenTasks/week2/T2_divisibleBy_3_5_15_JN.java:5]: This was my mistake by not updating the .txt file appropriately, will be sorted with future tasks

File: src/JenTasks/week2/T3_consecutiveNumbers_JN.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (cbanavi) [reply src/JenTasks/week2/T3_consecutiveNumbers_JN.java:4]: This was my mistake by not updating the .txt file appropriately, will be sorted with future tasks

**Pull request merged**